app/resources/start.html.template

```diff
@@ -29,31 +29,33 @@
 
     <div class="main">
       <div class="row">
-        {# Following onclick is to make the whole button clickable. #}
-        <div class="action-outer start"
-          {% if not env.amp %}
-            onclick="location.href = '{{seek_url}}';" style="cursor: pointer;"
-          {% endif %}
-        >
-          <div class="action-inner">
-            <a href="{{seek_url}}" tabindex="1">
-              {% trans "I'm looking for someone" %}
-            </a>
+        <div class="action-wrapper">
```

Comment:
Why did you introduce a new div? Can't you apply the styles for `row` instead?

Reply:
I couldn't find a way to do it without a wrapper div around the two buttons to center them together. If I just apply all the rules to the row class, it ends up left-aligned instead of centered.

Reply:
OK, then it's fine, but just curious, did you find the reason why it ended up left-aligned? Maybe due to some CSS rule conflict?